.github/workflows/cluster_endtoend_vtctlbackup_sharded_clustertest_heavy.yml

```diff
@@ -93,15 +93,16 @@ jobs:
       if: steps.skip-workflow.outputs.skip-workflow == 'false' && steps.changes.outputs.end_to_end == 'true'
       run: |
         
-        # Get key to latest MySQL repo
-        sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys A8D3785C
-        # Setup MySQL 8.0
-        wget -c https://dev.mysql.com/get/mysql-apt-config_0.8.32-1_all.deb
-        echo mysql-apt-config mysql-apt-config/select-server select mysql-8.0 | sudo debconf-set-selections
-        sudo DEBIAN_FRONTEND="noninteractive" dpkg -i mysql-apt-config*
+        # Setup Percona Server for MySQL 8.0
         sudo apt-get -qq update
+        sudo apt-get -qq install -y lsb-release gnupg2 curl
+        wget https://repo.percona.com/apt/percona-release_latest.$(lsb_release -sc)_all.deb
+        sudo DEBIAN_FRONTEND="noninteractive" dpkg -i percona-release_latest.$(lsb_release -sc)_all.deb
+        sudo percona-release setup ps80
+        sudo apt-get -qq update
+
         # Install everything else we need, and configure
-        sudo apt-get -qq install -y mysql-server mysql-client make unzip g++ etcd curl git wget eatmydata xz-utils libncurses5
+        sudo apt-get -qq install -y percona-server-server percona-server-client make unzip g++ etcd git wget eatmydata xz-utils libncurses5
```

Comment:
With this change all tests in vtctlbackup package will run with percona, not vanilla MySQL. @shlomi-noach / @mattlord is that acceptable?

Reply:
Wondering the same. We primarily test on vanilla MySQL except when Xtrabackup is involved. We must continue to ensure we test on vanilla MySQL.
If this test now needs to be able to support both MySQL and Percona Server at the same time, then possibly create a new test.

The `backup_pitr` family of tests are a convenient place to start. These are relatively quick tests that can run multiple backups/restores. If we need to introduce a new CI endtoend test, I'd suggest using that. `cluster_endtoend_vtctlbackup_sharded_clustertest_heavy.yml` is... heavy.

Reply:
I should add that as of now `go/test/endtoend/backup/vtctlbackup` tests builtin backups, and there are two other packages `go/test/endtoend/backup/xtrabackup` and `.../xtrabackup_xbstream` which test xtrabackup. Maybe the new test code should go into `xtrabackup` package instead of `vtctlbackup`. Then the changes to this workflow file can be reverted.